src/language/providers/completionItemCache.ts

```diff
@@ -0,0 +1,17 @@
+import { CompletionItem } from "vscode";
+import LRU from "lru-cache";
+
+export let updateCache: Set<string> = new Set<string>();
```

Comment:
I think an improved name could be used for more clarity. Perhaps `cacheForUpdate` or `changedCache`. It wasn't clear to me at first what the goal of this Set was.